templates/addons/cluster-autoscaler/cluster-autoscaler.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         app: ${CLUSTER_NAME}-cluster-autoscaler
     spec:
       containers:
-        - image: registry.k8s.io/autoscaling/cluster-autoscaler:${CLUSTER_AUTOSCALER_VERSION:=${KUBERNETES_VERSION%.*}.0}
+        - image: registry.k8s.io/autoscaling/cluster-autoscaler:${CLUSTER_AUTOSCALER_VERSION:=v1.29.4}
```

Comment:
Instead of hardcoding the version here, can we set the environment variable `CLUSTER_AUTOSCALER_VERSION` to `v1.29.4` when running the e2e tests?

Reply:
Sure, I'll do that

Reply:
I've added this as an environment variable in the chainsaw-test.yaml